web/webserver.go

```diff
@@ -50,3 +52,17 @@ func (s *QuickFeedService) RegisterRouter(tm *auth.TokenManager, authConfig *oau
 
 	return router
 }
+
+// controller is a wrapper for the QuickFeedService handler that sets a write deadline for the submission stream.
+// TODO: Remove this when connect-go finally supports deadlines.
```

Comment:
Once we merge this PR, we should add an issue to remove it again once the referenced issue has been fixed in `connectrpc`.